template/public/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app and Hipo React TypeScript template" />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Comment:
Is this logo the default react logo with atom core and the orbitals? 😄  
we may replace this with hipo logo, I think I see in on of the components.

Reply:
Yeah I didn't change it because it would require me to find a high quality image and resize it, so I didn't want to spend too much time on it. 😄  I think it can stay for now, we change it on every project anyway

Reply:
I used this in the previous starter template. We can convert to PNG and include it here too 😄 
https://github.com/Hipo/hipo-frontend-project-starter/blob/master/src/core/ui/icons/hipo-logo.svg

Reply:
created an issue https://github.com/Hipo/cra-template-hipo-typescript/issues/5